src/vaadin-select.html

```diff
@@ -177,15 +177,13 @@
        * @mixes Vaadin.ElementMixin
        * @mixes Vaadin.ControlStateMixin
        * @mixes Vaadin.ThemableMixin
-       * @mixes Vaadin.ThemePropertyMixin
        * @demo demo/index.html
        */
       class SelectElement extends
         Vaadin.ElementMixin(
           Vaadin.ControlStateMixin(
             Vaadin.ThemableMixin(
-              Vaadin.ThemePropertyMixin(
-                Polymer.mixinBehaviors(Polymer.IronResizableBehavior, Polymer.Element))))) {
+              Polymer.mixinBehaviors(Polymer.IronResizableBehavior, Polymer.Element)))) {
```

Comment:
Now `Vaadin.ThemableMixin` extends `Vaadin.ThemePropertyMixin` so there is no need to use it here.